bootstrap.go

```diff
@@ -561,6 +567,10 @@ func upgradeToVer17(s Session) {
 	doReentrantDDL(s, "ALTER TABLE mysql.user MODIFY User CHAR(32)")
 }
 
+func upgradeToVer18(s Session) {
+	doReentrantDDL(s, "ALTER TABLE mysql.stats_histograms ADD COLUMN `avg_col_size` double", infoschema.ErrColumnExists)
```

Comment:
If we upgrade an old cluster, there may be many existing tables, then the value of the new `avg_col_size` would default to NULL, to avoid this we can set a default value for this column like `8`.

Reply:
We could add a version for stats meta. So we could know that the current version is lack of avg-col-size.

Reply:
This is a temporary effect during the upgrade.
In the future, this column is never NULL and we don't need to check if it is NULL.